### ISSUE TITLE
Managed Identity credential support for BlobStorage

### ISSUE THREAD

**Issue by georgeOsdDev** [microsoft/botbuilder-python#2194]
**Is your feature request related to a problem? Please describe.**

Is it possible to using BlobStorage with ManagedId credential?

https://learn.microsoft.com/en-us/azure/bot-service/bot-builder-howto-v4-storage?view=azure-bot-service-4.0&tabs=python#using-blob-storage

It seems to be only connectionString or Key is supported.

https://github.com/microsoft/botbuilder-python/blob/main/libraries/botbuilder-azure/botbuilder/azure/blob_storage.py#L23

**Describe the solution you'd like**

Add support to use creadential.

**Describe alternatives you've considered**

N/A

**Additional context**

.Net and JavaScript SDK seems to supporting credential instead of connectionString.

https://learn.microsoft.com/en-us/dotnet/api/microsoft.bot.builder.azure.blobs.blobsstorage?view=botbuilder-dotnet-stable
https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-azure-blobs/src/blobsStorage.ts#L57

**Comment by jrudley** [comment]
I don't believe this works for JavaScript. Keeps throwing 401's for me. I can get the MI to work with OpenAI just fine.

**Comment by tracyboehrer** [comment]
This issue will not be addressed prior to Bot Framework SDK deprecation EOM December 2025.
